tests/ard_reprocessed_l1s/test_ard_reprocessed_l1s.py

```diff
@@ -156,34 +163,40 @@ def test_move_blocked(set_up_dirs_and_db):
     # blocking_ard_zip_path the file doesn't matter...I hope
     blocked_scenes = [
         {
-            "blocking_ard_id": "d9a499d1-1abd-4ed1-8411-d584ca45de25",
+            "blocking_ard_id": ard_id_06_27,
             "blocked_l1_zip_path": "not used",
-            "blocking_ard_zip_path": old_fname_06_27,
+            "blocking_ard_path": old_yaml_fname_06_27,
         }
     ]
     l1_zips, uuids2archive = move_blocked(
         blocked_scenes, current_base_path.resolve(), new_base_path.resolve()
     )
 
     # Assert the dir has been moved
-    assert os.path.isfile(fname_06_27) == True
+    assert os.path.isfile(yaml_fname_06_27)
     assert len(l1_zips) == 1
     assert len(uuids2archive) == 1
+    dc = datacube.Datacube(
+        app="test_ard_reprocessed_l1s", config=str(os.getenv("DATACUBE_CONFIG_PATH"))
+    )
+    ard_dataset = dc.index.datasets.get(ard_id_06_27)
+    local_path = Path(ard_dataset.local_path).resolve()
+    assert str(local_path) == str(yaml_fname_06_27)
```

Comment:
i would recommend that we put a message describing the objective of the test. 
 This allows for our tests to be more descriptive especially when they are run,

 `assert <test condition>, <string describing test's objective>`